src/libponyc/pass/docgen.c

```diff
@@ -543,7 +543,7 @@ static void doc_fields(docgen_t* docgen, docgen_opt_t* docgen_opt,
     }
 
     fprintf(docgen->type_file, "### %s %s: ", ftype, name);
-    doc_type(docgen, docgen_opt, type, true, true);
+    doc_type(docgen, docgen_opt, type, true, false);
```

Comment:
the last parameter is "is it ok the break the lines". It isn't in the case @redvers identified.